python/cudf/cudf/api/types.py

```diff
@@ -27,6 +27,55 @@
 )
 
 
+def dtype(arbitrary):
+    """
+    Return the cuDF-supported dtype corresponding to `arbitrary`.
+
+    Inputs
+    ------
```

Comment:
Is `Inputs` also recognized by the numpy docstyle? I've never seen it but that might just be my own blind spot.
```suggestion
    Parameters
    ------------
```

Reply:
Probably not.. not sure what I was thinking here :)